src/control/lib/hardware/sysfs/provider.go

```diff
@@ -604,13 +603,29 @@ func (s *Provider) IsIOMMUEnabled() (bool, error) {
 	return err == nil && len(dmars) > 0, nil
 }
 
-const (
-	// From linux syscall table.
-	PRCTL_SYSCALL = 157
-
-	// See /usr/include/linux/prctl.h
-	PR_GET_THP_DISABLE = 42
-)
+//const (
+//	// From linux syscall table.
+//	PRCTL_SYSCALL = 157
+//
+//	// See /usr/include/linux/prctl.h
+//	PR_GET_THP_DISABLE = 42
+//)
+//
+//// IsTEPEnabled checks whether transparent hugepages is enabled by interrogating sysfs and
+//// implements the THPDetector interface on sysfs provider.
+//func (s *Provider) IsTHPEnabled() (bool, error) {
+//	if s == nil {
+//		return false, errors.New("sysfs provider is nil")
+//	}
+//
+//	state, _, errno := syscall.RawSyscall6(uintptr(PRCTL_SYSCALL), uintptr(PR_GET_THP_DISABLE),
+//		0, 0, 0, 0, 0)
+//	if errno != 0 {
+//		return false, errors.Wrap(errno, "failed get transparent hugepages disable status")
+//	}
+//
+//	return state != 1, nil
+//}
 
 // IsTEPEnabled checks whether transparent hugepages is enabled by interrogating sysfs and
```

Comment:
Just noticed another typo
```suggestion
// IsTHPEnabled checks whether transparent hugepages is enabled by interrogating sysfs and
```

Reply:
done